environment.yml

```diff
@@ -17,6 +17,7 @@ dependencies:
     - jinja2
     - redis
     - redis-py
+    - tika=1.8.7
```

Comment:
@chrismattmann - Did you test this locally?  This tries to use the tika 1.8.7 package from binstar, which I'm guessing hasn't been uploaded yet.